docs/running/cloud/cloud.rst

```diff
@@ -0,0 +1,60 @@
+
+.. _cloudOverview:
+
+Running Toil in the Cloud
+=========================
+
+Toil can use cloud platforms in several different ways.
+
+.. _cloudJobStore:
+
+Storage (job store)
+-------------------
+
+Toil can make use of cloud storage such as Azure, AWS, or Google buckets to take of care of storage needs.
+
+This is useful when running Toil in single machine mode on any cloud platform since it allows you to
+make use of their integrated storage systems.
+
+.. FIXME add link to batch system overview for single machine mode
+
```

Comment:
It sort of cannot yet since we don't have a specific place where we explain the single machine batchsystem and/or compare / explain all of the other batchsystems. In other words, the batchsystem section needs an overhaul before this can adequately be fixed.